modules/imgproc/perf/perf_houghCircles.cpp

```diff
@@ -0,0 +1,60 @@
+#include "perf_precomp.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgproc/types_c.h"
+
+using namespace std;
+using namespace cv;
+using namespace perf;
+
+PERF_TEST(Basic, PerfHoughCircles)
```

Comment:
`PerfHoughCircles` should be the first argument, `Basic` should be the second one like that:
`PERF_TEST(test_case, specific_test)`

Reply:
I thought that was the right way to do it, but I got this error when I tried to run the tests: 
> Failed
> All tests in the same test case must use the same test fixture
> class.  However, in test case PerfHoughCircles,
> you defined test Basic and test ManySmallCircles
> using two different test fixture classes.  This can happen if
> the two classes are from different namespaces or translation
> units and have the same name.  You should probably rename one
> of the classes to put the tests into different test cases.

Maybe the problem is because I am using the `PERF_TEST` macro. I will look into it further.

Reply:
I was assuming `PERF_TEST` works like the googletest `TEST` macro. Unfortunately it only allows one test per test case. I tried `PERF_TEST_F` and it had the same limitation. I worked around it by changin the test case names to "PerfHoughCircles" and "PerfHoughCircles2".